libcudacxx/test/libcudacxx/std/numerics/complex.number/complex.transcendentals/pow_complex_complex.pass.cpp

```diff
@@ -60,7 +63,10 @@ __host__ __device__ void test_edges()
       }
       else
       {
-        assert(imag(r) == imag(z));
+        if(imag(r) != imag(z))
+        {
+          is_about(imag(r), imag(z));
+        }
```

Comment:
Ditto 
```suggestion
        is_about(imag(r), imag(z));
```

Reply:
Forgot to comment on using `is_about` unconditionally. IIRC, when real==imag, `is_about` ends up dividing by zero for some of the test cases. So, your suggestion makes sense, but `is_about` needs to be fixed first. I'll take a look.